### PR TITLE
feature/FIS-676 Map accessibility changes

### PR DESCRIPTION
**Pull request by danleech-defra**
[PI2: Mapping amendments for Beta](https://eaflood.atlassian.net/browse/FIS-676)

Changes in response to DAC Accessibility Audit. All client side with no additional requests to server.

### CODE REVIEW

File: server/src/sass/components/map/_map-key.scss [hunk 1]
```diff
@@ -107,6 +121,53 @@
   }
 }
 
+// Attribution
```
Comment (teddmason) [inline server/src/sass/components/map/_map-key.scss:124]: @danleech-defra Do you need this chunk of commented css left in?

File: server/src/sass/components/map/_ol.scss [hunk 2]
```diff
@@ -276,4 +276,4 @@
 .ol-overviewmap .ol-overviewmap-box:hover {
     cursor: move;
 }
-*/
+*/
```
Comment (teddmason) [inline server/src/sass/components/map/_ol.scss:279]: I know you haven't done anything to this file either, but there is also a huge amount of commented out lines, are they needed?

File: server/src/templates/description-live.html [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{% if model.numFeatures > 9 %}
```
Comment (teddmason) [inline server/src/templates/description-live.html:1]: The details of this need adding to the acceptance criteria, for scrutiny and testing

File: server/src/js/components/map/container.js [hunk 4]
```diff
@@ -76,116 +75,174 @@ window.flood.maps.MapContainer = function MapContainer (mapId, options) {
...
 
-  // Create viewport keyboard access tooltip
-  const tooltipElement = document.createElement('div')
-  tooltipElement.className = 'defra-map-tooltip'
-  tooltipElement.id = 'tooltip'
-  tooltipElement.setAttribute('role', 'tooltip')
-  tooltipElement.hidden = true
-  tooltipElement.innerHTML = 'Keyboard access guidelines'
-  viewport.appendChild(tooltipElement)
+  // Create the open key button
```
Comment (teddmason) [inline server/src/js/components/map/container.js:116]: Do you need this commented code leaving in?

**Comment by danleech-defra** [comment]
Unnecessary commented out code removed as per suggestions above and additional including _map.scss. Acceptance criteria updated to reflect work done.

**Review by teddmason — APPROVED** [review]

**Pull request merged**